### ISSUE TITLE
Unable to run multiple-datasources as-is on local Spring Cloud Data Flow installation

### ISSUE THREAD

**Issue by fysiskhund** [spring-cloud/spring-cloud-task#690]
Hi,
I cloned the repo to my hard disk and opened the multiple-datasources example with IntelliJ Idea. 
Without altering the source code, I ran mvn package to produce a *.jar.

I then registered the jar as an app using the Spring shell with this command:
```
app register --type task --name multiple-datasources --uri file://root/scdf/multiple-datasources-2.3.0-SNAPSHOT.jar
```
I created the task definition as shown:
<img width="871" alt="Screenshot 2020-07-13 at 21 44 25" src="https://user-images.githubusercontent.com/8021432/88238150-833fe080-cc46-11ea-8f28-f4b85636c2f0.png">

And tried to run it. Unfortunately, SCDF does not record start or stop times for the task execution, and looking into the logs vía the shell, I found the following relevant messages:

```
2020-07-22 23:00:59.298 ERROR 140 --- [           main] o.s.c.t.listener.TaskLifecycleListener   : An event to end a task has been received for a task that has not yet started.
2020-07-22 23:00:59.298  WARN 140 --- [           main] s.c.a.AnnotationConfigApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Failed to start bean 'taskLifecycleListener'; nested exception is java.lang.IllegalArgumentException: Invalid TaskExecution, ID 52 not found
```

I've looked at similar issues, but am unable to understand what I'm doing wrong. Moreover I was hoping that the example would just run "out of the box". What am I missing?

Thanks!

**Comment by cppwfs** [comment]
In this case we can add a profile to the sample that will allow a user to work with the sample standalone (default) and a another profile for a qualified datasource (dataflow) like the example here: https://github.com/cppwfs/multidbpropsample/blob/master/README.adoc

**Comment by cppwfs** [comment]
We have Resolved this issue with Issue #727 and will be available in the 2.3.0 release of Spring Cloud Task. Thanks for raising the issue.
